layout_padded/layout_padded.md

```diff
@@ -2651,18 +2675,35 @@ and where `s_dynamic` is the product of all values `extent(k)` for `k` in the ra
 
 [1]{.pnum} *Returns:*
 
-   * [1.1]{.pnum} `submdspan_mapping_result{*this, 0}`, if `Extents::rank() == 0` is `true`;
+   * [1.1]{.pnum} `submdspan_mapping_result{*this, 0}`, if _`rank_`_` == 0` is `true`;
+
+   * [1.2]{.pnum} otherwise, `submdspan_mapping_result{layout_right::mapping(sub_ext), offset}`, if
+
+      * _`rank_`_`==1` is `true`; or
 
-   * [1.2]{.pnum} otherwise, `submdspan_mapping_result{layout_right_padded<padding_value>::mapping(sub_ext), offset}`, if `Extents::rank() == 1` is `true`;
+      * `SubExtents::rank()==0` is `true`;
```

Comment:
Should we consider using this two-bullet-point format for all the analogous cases above?  I know LWG doesn't really like two bullet points, but we should at least have the same number of numbered paragraphs (one, in this case: 1.2) for all the cases.  I'll suggest appropriate changes above.

Reply:
Actually, never mind; I see why we might like to do this differently for the other layouts.